### PR TITLE
feat: add flow for sweeping funds from a V1 BTC wallet

### PR DESCRIPTION
**Pull request by rushilbg**
feat: add flow for sweeping funds from a V1 BTC wallet

Given a list of unspents to sweep.

BTC-1224

Ended up creating 2 full signed transactions using this flow for my v1safe wallet, Redash query below:
```
{
    "collection": "wallettx",
    "query": {
        "walletId": "2N85TyC2JdbBpanfme6w8XNQaSr5wcc9Ry9"
    },
    "limit": 2,
    "sort": [{
        "name": "createdDate",
        "direction": -1
    }]
}
```
Screenshot 2024-07-17 at 5 58 11 PM

WRW Flow
Screenshot 2024-07-15 at 2 42 02 PM
Screenshot 2024-07-15 at 2 42 22 PM
Screenshot 2024-07-15 at 2 43 05 PM
Screenshot 2024-07-15 at 2 43 23 PM
Screenshot 2024-07-15 at 2 43 35 PM
Screenshot 2024-07-17 at 3 05 31 PM
Screenshot 2024-07-17 at 3 05 51 PM
Screenshot 2024-07-17 at 4 23 42 PM

### CODE REVIEW

File: electron/preload/index.ts [hunk 1]
```diff
@@ -46,6 +46,7 @@ type Commands = {
       | createDotBroadcastableSweepTransactionParameters
       | createSolBroadcastableSweepTransactionParameters
   ): Promise<Error | BroadcastableSweepTransaction>;
+  sweepV1(coin: string, parameters);
```
Comment (OttoAllmendinger) [inline electron/preload/index.ts:50]: can we come up with a better type here?

File: src/containers/V1BtcSweep/index.tsx [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+              showSaveDialogData.filePath,
+              JSON.stringify(result, null, 2),
+              { encoding: 'utf8' }
+            );
+            navigate(
+              `/${environment}/v1btc-sweep/${coin}/success`
+            );
+          } catch (error) {
+            if (error instanceof Error) {
+              setFieldError('unspents', error.message);
```
Comment (OttoAllmendinger) [inline src/containers/V1BtcSweep/index.tsx:74]: why this field?

**Review by OttoAllmendinger — CHANGES_REQUESTED** [review]

File: src/containers/V1BtcSweep/index.tsx (hunk 2 above)
Comment (rushilbg) [reply src/containers/V1BtcSweep/index.tsx:74]: Remnant, will update.

**Review by OttoAllmendinger — APPROVED** [review]
issues to be fixed in follow up PR

**Pull request merged**